src/models/authentication.ts

```diff
@@ -0,0 +1,16 @@
+import { GenerateToken } from "../utils/generateToken";
+
+export class Authentication {
+
+    id?: number;
+    token: string;
+    email: string;
+    createdAt: Date;
+
+    constructor(email: string) {
```

Comment:
Use the same approach that you have in User, receive attributes to populate the authentication easily when you get from the database.

be aware of the undefined possibilities; for instance, if there is no token in attributes, generate it, something like: `this.token = attributes.token || GenerateToken.generateToken(50)`